services/accumulo-utils/pom.xml

```diff
@@ -0,0 +1,100 @@
+    <version>1.0-SNAPSHOT</version>
+    <properties>
+        <version.accumulo>1.9.2</version.accumulo>
+        <version.authorization-api>1.3</version.authorization-api>
+    </properties>
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>gov.nsa.datawave.microservice</groupId>
+                <artifactId>authorization-api</artifactId>
+                <version>${version.authorization-api}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.apache.accumulo</groupId>
+                <artifactId>accumulo-core</artifactId>
+                <version>${version.accumulo}</version>
+                <exclusions>
+                    <exclusion>
+                        <artifactId>hadoop-hdfs</artifactId>
+                        <groupId>org.apache.hadoop</groupId>
+                    </exclusion>
+                    <exclusion>
+                        <artifactId>hadoop-mapreduce-client-app</artifactId>
+                        <groupId>org.apache.hadoop</groupId>
+                    </exclusion>
+                    <exclusion>
+                        <artifactId>hadoop-mapreduce-client-core</artifactId>
+                        <groupId>org.apache.hadoop</groupId>
+                    </exclusion>
+                    <exclusion>
+                        <artifactId>hadoop-mapreduce-client-jobclient</artifactId>
+                        <groupId>org.apache.hadoop</groupId>
+                    </exclusion>
+                    <exclusion>
+                        <artifactId>hadoop-yarn-api</artifactId>
+                        <groupId>org.apache.hadoop</groupId>
+                    </exclusion>
+                    <exclusion>
+                        <artifactId>*</artifactId>
+                        <groupId>org.slf4j</groupId>
+                    </exclusion>
+                    <exclusion>
+                        <artifactId>commons-logging</artifactId>
+                        <groupId>commons-logging</groupId>
+                    </exclusion>
+                    <exclusion>
+                        <artifactId>junit</artifactId>
+                        <groupId>junit</groupId>
+                    </exclusion>
+                </exclusions>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+    <dependencies>
+        <dependency>
+            <groupId>gov.nsa.datawave.microservice</groupId>
+            <artifactId>authorization-api</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.accumulo</groupId>
+            <artifactId>accumulo-core</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-jcl</artifactId>
```

Comment:
Didn't notice anything in the test class that would use this. Did I miss it, or can this be removed?

Reply:
It was just to work around the missing runtime dependency on ```org.apache.commons.logging.LogFactory```

E.g., without that on the classpath you get the following during the test...
```
Caused by: java.lang.NoClassDefFoundError: org/apache/commons/logging/LogFactory
        at org.apache.hadoop.io.FastByteComparisons.<clinit>(FastByteComparisons.java:39)
        at org.apache.hadoop.io.WritableComparator.compareBytes(WritableComparator.java:187)
        at org.apache.accumulo.core.data.ByteSequence.compareTo(ByteSequence.java:116)
        at org.apache.accumulo.core.data.ByteSequence.compareTo(ByteSequence.java:24)
        at java.util.TreeMap.compare(TreeMap.java:1294)
        at java.util.TreeMap.put(TreeMap.java:538)
        at java.util.TreeSet.add(TreeSet.java:255)
        at java.util.AbstractCollection.addAll(AbstractCollection.java:344)
        at java.util.TreeSet.addAll(TreeSet.java:312)
        at java.util.TreeSet.<init>(TreeSet.java:160)
        at org.apache.accumulo.core.security.Authorizations.checkAuths(Authorizations.java:91)
```

Reply:
Yeah, our spring-boot:2.0.4.RELEASE depends on spring-core:5.0.8.RELEASE, so it seems like adding spring-jcl:5.0.8.RELEASE with runtime scope *should* be safe.

I've definitely experienced microservice start up failures in my own environment due to transitive dependency on older log4j artifacts, which aren't compatible with spring-boot 2.x. Wasn't very fun to troubleshoot

Reply:
Made a few pom updates, including adding runtime scope to spring-jcl. I had also noticed that I was getting commons-lang 2.x transitively from acccumulo-core, and that we were using commons-lang3 elsewhere. So, I updated for that as well